torch/_torch_docs.py

```diff
@@ -2579,15 +2589,18 @@ def merge_dicts(*dicts):
            r"""
 vdot(input, other, *, out=None) -> Tensor
 
-Computes the dot product (inner product) of two tensors. The vdot(a, b) function
+Computes the dot product (inner product) of two 1D tensors. The vdot(a, b) function
```

Comment:
(inner product) reference would be removed here, too, to be parallel with the above.